tei/la-estatua-de-prometeo.xml

```diff
@@ -5578,10 +5674,10 @@
             <l>en un generoso pecho</l>
             <l>como del desprecio mismo</l>
             <l>hacer yo misma desprecio.</l>
-            <stage>Bosque.</stage>
           </sp>
+          <stage>Bosque</stage>
```

Comment:
I am not sure this is the correct word. At least in context, it makes little sense.

Reply:
weird... need to look into detail

Reply:
I looked at the manuscript and yes it says "Bosque". I guess it means that the decoration has to change to "Bosque"?